### ISSUE TITLE
Panel hides words

### ISSUE THREAD

**Issue by jidanni** [osm-fr/infrastructure#471]
On https://layers.openstreetmap.fr/
there are some words hidden by the panel.
![20230704T133804](https://github.com/osm-fr/infrastructure/assets/1230959/4ce17204-3c89-4ba4-b32e-97bfc01b463e)

**Comment by Marc-marc-marc** [comment]
yes the list is too long right now and/or Permalink/Edit should be moved elsewhere on the page
